schema_salad/metaschema/salad.md

```diff
@@ -256,7 +256,8 @@ rules:
   field.
 
   * If the value of `jsonldPredicate` is an object, and that
-  object contains the field `_type` with the value `@id`, the field is a
+  object contains the field `_type` with the value `@id` and
+  the field `identity` with the value `false`, the field is a
```

Comment:
Should say identity is unspecified or false.

Reply:
> However there are also fields whose values have the "identity" property.

Can we still call them `link fields`?
If so, the following might be more appropriate fix for it.

```markdown
If the value of `jsonldPredicate` is an object, and that object contains the field `_type` with the value `@id`, the field is a link field.
If the field `jsonldPredicate` also has the field `identity` with the value `true`, the field is resolved with [identifier resolution](#Identifier_resolution).
Otherwise it is resolved with [link resolution](#Link_resolution).
```

What do you think?

Reply:
ping!

Reply:
ping!

Reply:
yes, that is much clearer!